src/ORM/DataObject.php

```diff
@@ -2692,6 +2692,16 @@ public function getViewerTemplates($suffix = '')
         return SSViewer::get_templates_by_class(static::class, $suffix, $this->baseClass());
     }
 
+    /**
+     * Get the link for editing this record in the CMS.
+     */
+    public function CMSEditLink(): ?string
```

Comment:
```suggestion
    public function getCMSEditLink(): ?string
```
So matches getCMSFields(), getCMSCompositeValidator() etc

Reply:
Since there's probably a million things that already use CMSEditLink(), should probably also add this, though don't remove in CMS 6

```php
/**
 * Alias for getCMSEditLink() for backwards-compatibility
 * @deprecated ... use getCMSEditLink() instead
 */
public function CMSEditLink(): ?string
{
    // Deprecation ... use getCMSEditLink() instead
    return $this->getCMSEditLink();
}
```

Reply:
`CMSEditLink` is more like `Link` than it is like `getCMSFields`.... but there's no harm in doing this the way you've proposed so I'll do that.

Reply:
Done - though I've intentionally _not_ strongly typed the new `CMSEditLink` method.
The whole point of adding that is so people aren't _forced_ to make a change in CMS 6. If we strongly typed this method, we would be forcing people to update their method signatures for that method.